cmd/ak/cmd/orgs/add_member.go

```diff
@@ -0,0 +1,40 @@
+package orgs
+
+import (
+	"fmt"
+
+	"github.com/spf13/cobra"
+
+	"go.autokitteh.dev/autokitteh/cmd/ak/common"
+	"go.autokitteh.dev/autokitteh/internal/resolver"
+)
+
+var addMemberCmd = common.StandardCommand(&cobra.Command{
+	Use:     "add-member <org id> <user id>",
```

Comment:
I think it's finally time to introduce context to the cli

context should include current org, current project, current token

so most commands won't require the basic args like org id and project id and users would be able to switch context easily

something like

```bash
# Create Context
ak context create --name "my context" --token my_token --server https://api.autokitteh.cloud # use default org , server is cloud by default, can be localhost by default or by setting --local instead of server

ak context create --name "my context" --token my_token --org org_id # use any org id allowed for token

ak context create --name "my context" --token my_token --org org_id --project project_id # use specific project, not sure this is required but can be useful

# List Context
ak context list

# Switch context
ak context use context_name
```